addons/im_livechat/views/res_users_views.xml

```diff
@@ -23,9 +29,14 @@
             <field name="inherit_id" ref="base.view_users_form"/>
             <field name="arch" type="xml">
                     <xpath expr="//group[@name='messaging']" position="after">
+                        <field name="has_access_livechat" readonly="1" invisible="1"/>
                         <group name="livechat" string="Livechat"
-                            invisible="share">
+                            invisible="share or not has_access_livechat">
```

Comment:
Why not just having `not has_access_livechat` in condition? If a `share` user has livechat access, something strange is going on.